packages/appsync-emulator-serverless/lambdaSource.js

```diff
@@ -39,8 +40,18 @@ const lambdaSource = async (
   );
   let child = null;
 
-  if (fnConfig.runtime && fnConfig.runtime.indexOf('python') >= 0) {
-    child = fork(PythonRunner, [], {
+  if (fnConfig.runtime && !fnConfig.runtime.includes('node')) {
+    let extHandlerMethod = '';
+    let runner = null;
+    if (fnConfig.runtime.indexOf('python') >= 0) {
```

Comment:
this is begging for a switch statement but we can refactor this if we add a third runtime (or make the runtimes generic enough that we're just invoking sam or similar)